app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::UsersController < ApplicationController
+	before_action :authenticate_user!
```

Comment:
Lets authenticate user in application controller and skip this action in the controllers where not required?